src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -684,7 +684,7 @@ public class StripePaymentService : IPaymentService
             // Retain original collection method
             var collectionMethod = sub.CollectionMethod;
 
-            var subResponse = await subscriptionService.UpdateAsync(sub.Id, new SubscriptionUpdateOptions
+            var subUpdateOptions = new SubscriptionUpdateOptions()
```

Comment:
remove empty parens `()` at end of object constructor when using simplified object initialization (`{ ... }`).